mecademic_description/urdf/meca_500_r3.ros2_control.xacro

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="meca_500_r3_ros2_control" params="
+    name
+    prefix
+    robot_ip
+    control_port
+    monitoring_port
+    velocity_timeout
+    velocity_limit_percent
+    acceleration_limit_percent
+    use_fake_hardware:=false
+    fake_sensor_commands:=false
+    initial_positions:=${dict(axis_1_joint=0.0,axis_2_joint=0.0,axis_3_joint=0.0,axis_4_joint=0.0,axis_5_joint=0.0,axis_6_joint=0.0)}">
+
+    <ros2_control name="${name}" type="system">
+      <hardware>
+        <xacro:if value="${use_fake_hardware}">
```

Comment:
What happens if fake hardware is true? What do `fake_sensor_commands` and `state_following_offset` do and what is a GenericSystem?